components/DogProfile/DogProfile.js

```diff
@@ -25,15 +25,17 @@ export default function DogProfile({ dogData }) {
           {inHeat && <p>läufig</p>}
         </StyledGenderSection>
         <StyledLabel>Alter: </StyledLabel>
-        <p>{age}</p>
+        <p>{age > 1 ? `${age} Jahre` : `1 Jahr oder jünger`}</p>
```

Comment:
I think here it's not a matter of truth or not but rather: if age is bigger thann 1 than it should be "Jahre" not Jahr. Or did I misunderstand you?

Reply:
@alexWaligorski Your approach is not wrong, because it works. I just wanted to show you another approach. 

**Let me try to explain it in more detail:**
_For example: In the form, the user selects the year of birth. If this is 2023, age is: 
`{... 
  age: 0;
... }`_
- `<p>{age ? ${age} Jahre:1 Jahr oder jünger}</p>`
-  This expression above checks if the value of age is `truthy` (i.e., **not a `falsy value`** like 0, null, undefined, etc.). 
- If it is `truthy`, it renders the string _"${age} Jahre"_ 
- If age is `falsy` (e.g., 0, null, or undefined), it renders the string _"1 Jahr oder jünger"_

So if the selected age is `0` it is a `falsy value` and the corresponding string will be rendered and you do not have to compare it with `> 1`. 
By the way comparison operators produce boolean values as well.

Reply:
@JessicaLoers Thanks so much for your explanation. Maybe I have a "Knoten im Kopf", because I think your solution would result in a string like this: "1 Jahre" , if age === 1 
This is, why I am using > 1 here.